.circleci/cancel-jobs.js

```diff
@@ -0,0 +1,34 @@
+const request = require('request');
```

Comment:
Can we use fetch?

Reply:
👍 We should be able to. I was basically just copying the example in some circle-ci docs.